protos/sources/vega/vega.proto

```diff
@@ -1245,6 +1247,22 @@ message LiquidityProviderFeeShare {
   string virtual_stake = 5;
 }
 
+// SLA performance for each liquidity provider
+message LiquidityProviderSLA {
+  // Liquidity provider party ID.
+  string party = 1;
+  // Indicates how often LP meets the commitment during the current epoch.
+  string current_epoch_fraction_of_time_on_book = 2;
+  // Indicates how often LP met the commitment in the previous epoch.
+  string last_epoch_fraction_of_time_on_book = 3;
+  // Indicates the fee penalty amount applied in the previous epoch.
+  string last_epoch_fee_penalty = 4;
+  // Shows the fee penalties from past epochs.
+  string last_epoch_bond_penalty = 5;
+  // Holds the fee penalties from past epochs.
```

Comment:
I don't really understand this string. Should it be something like "determines how the fee penalties from past epochs affect future fee revenue"?